SCF/Assignment3B/TestMarksheet.java

```diff
@@ -0,0 +1,124 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * 
+ * @author Nilesh Patel
+ * MarksheetMain Class used to check methods of 
+ * Marksheet Class. It take input from user console
+ * and get the result from Marksheet Class and print
+ * them on console
+ */
+public class TestMarksheet {
+
+	public static void main(String[] args)
+	{
+		Marksheet marksheet = new Marksheet();
+		Scanner userInput = new Scanner(System.in);
+		
+		//taking number of student input
+		System.out.println("Enter Number Of Student : ");
+		int numberOfStudent = userInput.nextInt();
```

Comment:
``` Add validation message```  if numberOfstudent is less than or equal to zero

Reply:
Done